runtime/src/proposals_configuration/playground.rs

```diff
@@ -61,7 +61,7 @@ pub(crate) fn funding_request_proposal() -> ProposalParameters<BlockNumber, Bala
 pub(crate) fn create_working_group_lead_opening_proposal(
 ) -> ProposalParameters<BlockNumber, Balance> {
     ProposalParameters {
-        voting_period: 200,
+        voting_period: 20,
```

Comment:
I don't think voting period length of the proposals adds to the delay, because the proposal should be approved instantly once the quorum+threshold is reached, so perhaps it's better to keep it intact.

Probably the main delay was the `council announcing period + referendum voting + referendum revealing period`, which are together `100 + 100 + 100 = 300 blocks (~1800 sec)` now, and were `300 + 100 + 100 = 500 blocks (~3000 sec)` before

```suggestion
        voting_period: 200,
```